RichText.hpp

```diff
@@ -22,6 +22,12 @@ typedef Rect<float> FloatRect;
 
 namespace sfe
 {
+struct RichColor
+{
+	sf::Color fill = sf::Color::White;
+	sf::Color outline = sf::Color::Transparent;
+	float thickness = 0.f;
```

Comment:
The only thing that comes to mind is something like "ColorProperties", but it's an awfully long name...

Reply:
Not sure if SFML uses short names, but it could be `ColorProps`. Anyway, the more I program in different languages, the less I get annoyed by long names. Maybe it isn't noticeable in the short term, but reading `ColorProperties` instead of `ColorProps` is easier to me.

Reply:
I'm not good at english at all but `TextStroke` sounds good!

Reply:
It was a suggestion from someone from the SFML discord.